Lib/idlelib/config-keys.def

```diff
@@ -57,6 +57,19 @@ toggle-tabs=<Alt-Key-t> <Meta-Key-t> <Alt-Key-T> <Meta-Key-T>
 change-indentwidth=<Alt-Key-u> <Meta-Key-u> <Alt-Key-U> <Meta-Key-U>
 del-word-left=<Control-Key-BackSpace>
 del-word-right=<Control-Key-Delete>
+autocomplete=<Key-Tab>
+try-open-completions=<KeyRelease-period> <KeyRelease-slash> <KeyRelease-backslash>
+force-open-completions=<Control-Key-space>
+expand-word=<Alt-Key-slash>
+try-open-calltip=<KeyRelease-parenleft>
+refresh-calltip=<KeyRelease-parenright> <KeyRelease-0>
+force-open-calltip=<Control-Key-backslash>
+toggle-code-context=<Alt-Key-3>
+format-paragraph=<Alt-Key-q>
+flash-paren=<Control-Key-0>
+paren-closed=<KeyRelease-parenright> <KeyRelease-bracketright> <KeyRelease-braceright>
+run-module=<Key-F5>
+check-module=<Alt-Key-x>
```

Comment:
Only current user-configurable 'event = keylist' pairs from [Xtension-cfgBindings] sections should be added.  These are

force-open-completions=<Control-Key-space>
expand-word=<Alt-Key-slash>
force-open-calltip=<Control-Key-backslash>
format-paragraph=<Alt-Key-q>
flash-paren=<Control-Key-0>
run-module=<Key-F5>
check-module=<Alt-Key-x>

The keys for autocomplete, try-open-completions, try-open-calltip, refresh-calltip, and paren-closed are not and should not be configurable.  Adding new shortcuts, such as for toggle-code-context, will be a separate issue.